tanner/web/templates/base.html

```diff
@@ -1,5 +1,28 @@
 <!DOCTYPE html>
 <html>
+<head>
+	<style>
+		.topright{
+			position: fixed;
+			right: 0;
+			top: 1;
+			width: 260px;
+		}
+	</style>
+	<div class="topright">
+			<a href="/" style="text-decoration:none;" >
+				Home
+			</a>
+			|
+			<a href="http://tanner.readthedocs.io/en/latest/" style="text-decoration:none;" >
+				Docs
+			</a>
+			|
+			<a href="/static/manual.pdf" style="text-decoration:none;" >
+				Commands-manual
```

Comment:
I suggest you to make a filters manual instead of commands, where you can list all available filters. Or we can keep in the docs and even delete this commands-manual page